### DISCUSSION TITLE
Import output from AuxKernel in Kernel

### DISCUSSION THREAD

**Discussion by ashishdhole** [idaholab/moose#26602] [Q&A Modules: Phase field]
I am working on Grain growth model with Solute drag pressure, which is a function of interface/grain Boundary Velocity. is there a way I can import the velocity at each _qp at each timestep (AuxKernel - GrainBoundaryVelocity) into the Kernel (ACGrGrPoly).
Thank you

**Comment by GiudGiud** [comment]
Hello

You can represent the velocity as a variable or as a material property. Those can then be retrieved inside a kernel.

See CoupledForce and MatDiffusion kernels for examples

Guillaume
